plugin/src/openshift/components/KialiController.tsx

```diff
@@ -131,16 +142,42 @@ class KialiControllerComponent extends React.Component<KialiControllerProps> {
         getJaegerInfoPromise
       ]);
 
-      this.props.setNamespaces(configs[0].data, new Date());
+      const namespaces = configs[0].data;
+      this.props.setNamespaces(namespaces, new Date());
       setServerConfig(configs[1].data);
       this.applyUIDefaults();
-      this.setState({ configLoaded: true });
+      this.setState({ configLoaded: true, namespaces });
     } catch (err) {
       console.error('Error loading kiali config', err);
     }
   };
 
-  private applyUIDefaults() {
+  private getKialiSecurityInfo = async (): Promise<void> => {
+    try {
+      const getMeshMTLSPromise = this.promises
+        .register('getMeshMTLS', API.getMeshTls())
+        .then(response => this.props.setMeshTlsStatus(response.data))
+        .catch(error => {
+          // User without namespaces can't have access to mTLS information. Reduce severity to info.
+          const informative = this.state.namespaces && this.state.namespaces.length < 1;
+          if (informative) {
+            AlertUtils.addError('Mesh-wide mTLS status feature disabled.', error, 'default', MessageType.INFO);
+          } else {
+            AlertUtils.addError('Error fetching Mesh-wide mTLS status.', error, 'default', MessageType.ERROR);
+          }
+        });
+
+      const getIstioCertsInfoPromise = this.promises
+        .register('getIstioCertsInfo', API.getIstioCertsInfo())
+        .then(response => this.props.setIstioCertsInfo(response.data));
+
+      await Promise.all([getMeshMTLSPromise, getIstioCertsInfoPromise]);
+    } catch (err) {
+      console.error('Error loading kiali security info', err);
```

Comment:
Shouldn't this be also added as an AlertUtils to warn the user?

Reply:
I think it is not worth to show to the user the global error message + specific ones. For me it is enough to show the specific ones to the user. Besides, I have noticed that AlertUtils error messages are not shown in OSSMC. I will create separate issue since this is not part of this PR.

Reply:
Well, that was for the ones that were not captured. I think it is better to show the specific error message, the only problem that I see, is if there is a general problem in the network or the access to the Kiali proxy and the user receives too many alerts.

Reply:
Yes, although on the other hand it shows to the user the feeling that something really wrong is happening to the application hahaha